tests/lean/run/1659.lean

```diff
@@ -1,19 +0,0 @@
-import data.hash_map
```

Comment:
This test was removed because it depends on some intimate details of the `hash_map` structure, and it wasn't clear that making a stub here would preserve the bugged behavior under test. If we really want to keep this bug test, we should probably try to make a better minimum working example using the old build or something.